master/definitions.xml

```diff
@@ -1219,6 +1219,7 @@
   <interface name='CanvasDrawPath' href='https://html.spec.whatwg.org/multipage/canvas.html#canvasdrawpath' />
 
   <!-- ... terms .......................................................... -->
+  <term name='Animation type' href='https://www.w3.org/TR/web-animations-1/#animating-properties'/>
```

Comment:
Done. SVG build system doesn't make it so easy to switch between ED and WD depending on the publication type as Bikeshed. Since the spec is in CR going to PR it seems saver to use TR links.